src/FormMigration/Steps/FormMeta.php

```diff
@@ -17,13 +18,20 @@ public function process()
         $oldFormId = $this->formV2->id;
         $newFormId = $this->formV3->id;
 
-        $oldFormMeta = give()->form_meta->get_meta($oldFormId);
+        $formMetaTable = DB::prefix('give_formmeta');
 
-        if ($oldFormMeta && is_array($oldFormMeta)) {
-            foreach ($oldFormMeta as $oldFormMetaKey => $oldFormMetaValue) {
-                $oldFormMetaValue = is_array($oldFormMetaValue) && count($oldFormMetaValue) === 1 ? $oldFormMetaValue[0] : $oldFormMetaValue;
-                give()->form_meta->update_meta($newFormId, $oldFormMetaKey, $oldFormMetaValue);
-            }
-        }
+        DB::query(
+            "
+                INSERT INTO $formMetaTable (form_id, meta_key, meta_value)
+                SELECT
+                    $newFormId,
+                    meta_key,
+                    meta_value
+                FROM
+                    $formMetaTable
+                WHERE
+                    form_id = $oldFormId
+                "
```

Comment:
Let's be good kids and use `DB::prepare` to insert the `$newFormId` and $oldFormId`. 🍭

Reply:
Added here: ddb4fb3e86a412f9c079df0d6bc6bd086d42babb